docs/es/budget.html

```diff
@@ -0,0 +1,84 @@
+﻿<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <title>Budgeting</title>
+    <link href="../manual.css" rel="stylesheet" type="text/css">
+    <script src="../js/get_param.js"></script>
+    <script src="../js/toc.js"></script>
+    <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js'></script>
+    <script src="../js/back_to_top.js"></script>
+</head>
+<body onload="generateTOC(document.getElementById('toc'));">
+    <header>
+        <nav><small><a id="go_home" href="index.html">Go to main help page</a></small></nav>
```

Comment:
```suggestion
        <nav><small><a id="go_home" href="../index.html?lang=es">Go to main help page</a></small></nav>
```

Reply:
Disagree. it should not be a language code.